docs/getting-started/peoplefinder/set-up-users.mdx

```diff
@@ -22,12 +22,12 @@ records from your tenant by disconnecting the `acmecorp` identity provider.
 
 Navigate to the “Connections” tab
 
-<img src="/connections-tab.png" width="500px" />
```

Comment:
sometimes the width is actually useful. You don't want a small image to fill the entire screen.